docs/Compilation/AutoInitialization.md

```diff
@@ -0,0 +1,534 @@
+<p></p> <!-- avoids duplicate title -->
+
+Automatic Initialization of Variables
```

Comment:
Should I adapt this document for the RM as well? Or just refer to it? Or have the RM replace this document?

Reply:
Excellent.